packages/shorebird_cli/lib/src/commands/patch/patch_ios_command.dart

```diff
@@ -307,6 +314,7 @@ Please re-run the release command for this version or create a new release.''');
         if (useLinker) {
           final (:exitCode, :linkPercentage) = await _runLinker(
             releaseArtifact: releaseArtifactFile,
+            dumpDebugInfo: dumpDebugInfo,
```

Comment:
I guess we don't need to inject this since we can access the flag from results directly within `_runLinker`?